build/uuid-redirects.json

```diff
@@ -54,6 +54,14 @@
     "Compendium.pf2e.classfeatures.Item.T9gUmwghw16itUAO": "Compendium.pf2e.classfeatures.Item.Catharsis Emotion",
     "Compendium.pf2e.classfeatures.Item.zFY2PlJTObXpcQW4": "Compendium.pf2e.classfeatures.Item.Catharsis Emotion",
     "Compendium.pf2e.equipment-effects.Item.5oYKYXAexr0vhx84": "Compendium.pf2e.other-effects.Item.Effect: Treat Disease",
+    "Compendium.pf2e.equipment-effects.Item.QuZ5frBMJF3gi7RY": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.7z1iY4AaNEAIKuAU": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.1mKjaWC65KWPuFR4": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.mi4Md1fB2XThCand": "Compendium.pf2e.equipment-effects.Item.Effect: Antidote",
+    "Compendium.pf2e.equipment-effects.Item.R106i7WCXvHLGMTu": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
+    "Compendium.pf2e.equipment-effects.Item.kkDbalYEavzRpYlp": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
+    "Compendium.pf2e.equipment-effects.Item.qVKrrKpTghgMIuGH": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
+    "Compendium.pf2e.equipment-effects.Item.HeRHBo2NaKy5IxhU": "Compendium.pf2e.equipment-effects.Item.Effect: Antiplague",
```

Comment:
Try to keep the file alphabetised. We'll probably conflict, but it's easy to resolve.

Reply:
OK, that should be set now.

Check when you get a moment.

Thanks!